packages/ui/src/components/forms/form/Form.vue

```diff
@@ -65,6 +65,10 @@ const props = defineProps({
   loadingText: {
     type: String,
     default: ""
+  },
+  titleClass: {
```

Comment:
do we still want to hold onto that even with the slot?

Reply:
I think its still useful cause if you just want to change the text color for example you don't have to make the whole slot can just add a little class

Reply:
ah fair, makes sense